framework/src/media/audio/audio_manager.c

```diff
@@ -1025,17 +1028,43 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 	return pcm_bytes_to_frames(g_audio_out_cards[g_actual_audio_out_card_id].pcm, bytes);
 }
 
-uint16_t get_max_audio_volume(void)
+audio_manager_result_t get_max_audio_gain(uint8_t *gain)
```

Comment:
I think gain is not neccessary please let me know reason why you added it?

Reply:
This function is deleted.